Dockerfile

```diff
@@ -48,4 +45,21 @@ COPY doc/datasets/popularize_churn.sql \
 ADD scripts/start.sh /
 ADD scripts/convert_markdown_into_ipynb.sh /
 
+# -----------------------------------------------------------------------------------
+# Above Steps Should be Cached for Each CI Build if Dockerfile is not Changed.
+# -----------------------------------------------------------------------------------
+
+# Build SQLFlow, copy sqlflow_submitter, convert tutorial markdown to ipython notebook
+COPY . ${GOPATH}/src/sqlflow.org/sqlflow
+RUN cd /go/src/sqlflow.org/sqlflow && \
+go generate ./... && \
+go get -t ./... && \
+go install -v ./... && \
+mv $GOPATH/bin/sqlflowserver /usr/local/bin && \
+mv $GOPATH/bin/repl /usr/local/bin && \
+cp -r $GOPATH/src/sqlflow.org/sqlflow/sql/python/sqlflow_submitter /miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/ && \
+cd / && \
+bash ${GOPATH}/src/sqlflow.org/sqlflow/scripts/convert_markdown_into_ipynb.sh && \
+rm -rf ${GOPATH}/src && rm -rf ${GOPATH}/bin
+
```

Comment:
Do we need to move `# 6. Install latest sqlflow_models for testing custom models` to Dockerfile? Because the models repo also changes frequently.